cmd/medius/common/registry.go

```diff
@@ -25,43 +25,50 @@ type Entry struct {
 var staticRegistry = []Entry{
 	{
 		Artifacts: []api.Artifact{
-			centos.New("8.4", nil),
+			centos.New("8.4", "x86_64", nil),
+			centos.New("8.4", "aarch64", nil),
 		},
 		UseForDocs: false,
 	},
 	{
 		Artifacts: []api.Artifact{
-			centos.New("7-2009", defaultEnvVariables("u1.small", "centos.7")),
+			centos.New("7-2009", "x86_64", defaultEnvVariables("u1.small", "centos.7")),
+			centos.New("7-2009", "aarch64", defaultEnvVariables("u1.small", "centos.7")),
 		},
 		UseForDocs: true,
 	},
 	{
 		Artifacts: []api.Artifact{
-			centosstream.New("9", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.small", "centos.stream9")),
+			centosstream.New("9", "x86_64", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.small", "centos.stream9")),
```

Comment:
I see that this is an example of user data. Personally I don't like to hide arguments inside functions - Because we had architecture hidden inside - Therefore these PR changes required now.

Reply:
This is slightly different as it's associated directly with the version being used. Anyway it's unrelated to your current PR but might be something I follow up with later if I get the time.

@0xFelix btw ^